src/main/java/frc/robot/constants/TrigonTalonFXConfig.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.constants;
```

Comment:
This entire class seems like a missunderstanding. Delete this and just give TrigonTalonFx a constructor that gets PIDCoefs

Reply:
I agree